src/utils/consts.ts

```diff
@@ -66,17 +67,59 @@ export enum CallableFunctions {
   GetUsersByEmail = 'getUsersByEmail'
 }
 
+export type IconName = 'AddNew' | 'Remove' | 'Waste';
+
+export const Icons: { [iconName in IconName]: string } = {
+  AddNew: require('@/assets/images/AddNew.svg'),
+  Remove: require('@/assets/images/Remove.svg'),
+  Waste: require('@/assets/images/Waste.svg')
+};
+
+export type FridgeAction = 'delete' | 'waste' | 'consume';
+
+export const fridgeActions: {
+  [actionName in FridgeAction]: { act: (product: Product) => {}; message: string };
+} = {
+  delete: {
+    act: p => p.removeFromStorage(),
+    message: 'Products were deleted'
+  },
+  waste: {
+    act: p => p.waste(),
+    message: 'Products were wasted'
+  },
+  consume: {
+    act: p => p.consume(),
+    message: 'Products were consumed'
+  }
+};
+
+export type ShoppingListAction = 'delete' | 'purchase';
+
+export const shoppingListActions: {
+  [actionName in ShoppingListAction]: { act: (product: Product) => {}; message: string };
+} = {
+  delete: {
+    act: p => p.removeFromShoppingList(),
+    message: 'Products were deleted'
+  },
+  purchase: {
+    act: p => p.purchase(),
+    message: 'Products were wasted'
```

Comment:
The message needs to be updated ('Products were purchased/moved to the storage')